src/api/structure/testing-unit.ts

```diff
@@ -155,6 +155,25 @@ export default abstract class TestingUnit extends BaseUnit {
         //@ts-ignore
         ChildClass.API_LIST = TestingUnit.API_LIST.concat(getDelegatedAPIList(ChildClass.prototype));
     }
+
+    public static initTestingUnit (ChildClass: unknown, ...initProps: unknown[]): Function {
```

Comment:
I think you can rename this method to `init` to consistent with `Fixture` and `Test` classes.